### PR TITLE
Fix Antimony

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
Fix for #45960 

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change (none)
- [x] Tested the program
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build antimony

File: pkgs/applications/graphics/antimony/default.nix [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
+, libGLU_combined, qtbase, ncurses
+, cmake, flex, lemon
+}:
 
 let
-  gitRev    = "020910c25614a3752383511ede5a1f5551a8bd39";
-  gitBranch = "master";
+  gitRev    = "60a58688e552f12501980c4bdab034ab0f2ba059";
+  gitBranch = "develop";
   gitTag    = "0.9.3";
```
Comment (Mic92) [inline pkgs/applications/graphics/antimony/default.nix]: `gitTag` can be removed for the time being.

Comment (rnhmjoj) [reply pkgs/applications/graphics/antimony/default.nix]: It's still used in line 39, I think it's used by the program to display the current version.

**Comment by xeji** [comment]
backported: d164d7eb897

**Pull request merged**